great_expectations/zep/type_lookup.py

```diff
@@ -22,24 +45,37 @@ class TypeLookup(
 
     def __init__(
         self,
-        __dict: Optional[Mapping[Hashable, Hashable]] = None,
+        __dict: Optional[Mapping[ValidTypes, ValidTypes]] = None,
         **kwargs: Hashable,
     ):
         __dict = __dict or {}
+        self._original_keys: Set[ValidTypes] = set()
```

Comment:
It looks like you skipped this one line when removing `_original_keys`.

Reply:
Good catch. Removing.